src/sentry/static/sentry/app/components/accordionButton.jsx

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+import {Flex} from 'grid-emotion';
+import Badge from 'app/components/badge';
+
+class AccordionButton extends React.Component {
+  static propTypes = {
+    onClick: PropTypes.func,
+    open: PropTypes.bool,
+    count: PropTypes.number,
+  };
+
+  render() {
+    const {style, className, count, onClick, open} = this.props;
+    return (
+      <StyledButton style={style} className={className} onClick={onClick} open={open}>
+        <Flex align="center" justify="space-between" w={1}>
+          <Flex align="center">
+            {this.props.children}
+            {count && <Badge style={{marginLeft: space(0.25)}} text={count} />}
+          </Flex>
+          <StyledInlineSvg src="icon-chevron-down" open={open} />
+        </Flex>
+      </StyledButton>
+    );
+  }
+}
+
+const StyledButton = styled('a')`
```

Comment:
Is it an `a` or `button`?

Reply:
it has to be an <a> because a button will cause forms to lose focus